spec/value_objects/ioj_reason_type_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe IojReasonType do
```

Comment:
Super tiny thing but might be worth having a little test for the `#justification_field_name` method, to cover all our bases 😉